core/assets/bundles/bundle_fr.properties

```
@@ -2295,10 +2295,10 @@ lst.setrate = Définit la vitesse d'exécution d'un processeur en instructions/t
 lst.fetch = Cherche les unités, noyaux, joueurs ou constructions par index. Commence à 0 et termine par le nombre retourné.
 lst.packcolor = Compresse les composants RGBA [0, 1] en un seul nombre pour les opérations de dessins ou les changements de règles.
 lst.setrule = Change une règle du jeu.
-lst.flushmessage = Affiche un message sur l'écran depuis la mémoire tampon de texte.\nLe message apparait après la fin du dernier.
+lst.flushmessage = Affiche un message sur l'écran depuis la mémoire tampon de texte.\nAttendra la fin du message précédent avant l'affichage du nouveau.
 lst.cutscene = Manipule la caméra du joueur.
-lst.setflag = Définit un drapeau global qui peut être lu par tous les processeurs.
-lst.getflag = Vérifie si un drapeau global est présent.
+lst.setflag = Définit une variable global qui peut être lu par tous les processeurs.
+lst.getflag = Vérifie si une variable global est présent.
```

```suggestion
lst.setflag = Définit une variable globale qui peut être lue par tous les processeurs.
lst.getflag = Vérifie si une variable globale est présente.
```
Attention les accords.